subworkflows/local/align_cellranger.nf

```diff
@@ -40,7 +41,21 @@ workflow CELLRANGER_ALIGN {
         )
         ch_versions = ch_versions.mix(CELLRANGER_COUNT.out.versions)
 
+        // rebuild out channel to be in compliance with what is required for h5ad conversion modules
+        // out channel comes without meta map from nf-core module
```

Comment:
Maybe easier to add meta map to output of module? We do this for most modules anyway, don't we?

Reply:
Fair enough! If you make a PR to the module, I can review&merge it.

Reply:
done in commit https://github.com/nf-core/scrnaseq/pull/117/commits/2edec15de7c6cb552728e27e974df9a31b8283bf